Sources/Logging/Strings/PurchaseStrings.swift

```diff
@@ -86,6 +86,7 @@ enum PurchaseStrings {
     case caching_presented_offering_identifier(offeringID: String, productID: String)
     case payment_queue_wrapper_delegate_call_sk1_enabled
     case restorepurchases_called_with_allow_sharing_appstore_account_false
+    case sk2_observer_mode_error_processing_transaction(error: Error)
```

Comment:
Nit: the `error` label isn't required since it doesn't add anything to the type:
```suggestion
    case sk2_observer_mode_error_processing_transaction(Error)
```